lib/components/SInputFileUpload.vue

```diff
@@ -121,8 +166,31 @@ function onRemove(index: number) {
         @change="onChange"
       >
       <SCard :mode="hasError ? 'danger' : undefined">
-        <SCardBlock class="header">
+        <SCardBlock v-if="droppable" class="drop-zone" ref="dropZoneEl">
+          <div class="drop-zone-box">
+            <STrans lang="en">
+              <div class="drop-zone-text">
+                <div>Drag & Drop files here</div>
+                <div>Or</div>
+              </div>
+              <div class="drop-zone-action">
+                <SButton size="mini" label="Select files" />
```

Comment:
missing handler

Reply:
Thanks fixed! But, the handler is added to the parent div so that the user can click the entire box 👀